### PR TITLE
fix: Added missing `NODE_VERSION` options in Dockerfile

### PR DESCRIPTION
**Pull request by mdsanima**
Now this is a properly configured file for Dev Container.

Only installing Python on the container, the Node is not installed, as was the case before.

#### Description

_Please explain the changes you made here._

#### Requirements / Checklist

- [x] Read the [Contributing Guidelines](https://github.com/ryanoasis/nerd-fonts/blob/-/contributing.md)
- [ ] Verified the license of any newly added font, glyph, or glyph set

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,9 @@
 ARG VARIANT=3
 FROM mcr.microsoft.com/vscode/devcontainers/python:${VARIANT}
 
+ARG NODE_VERSION="none"
+RUN if [ "${NODE_VERSION}" != "none" ]; then su vscode -c "umask 0002 && . /usr/local/share/nvm/nvm.sh && nvm install ${NODE_VERSION} 2>&1"; fi
```
Comment (Finii) [inline .devcontainer/Dockerfile:5]: You set `NODE_VERSION` explicitly to `none` and then you do some code when it is not `none` (which can never be the case). The `if` is always false.

**Review by Finii — COMMENTED** [review]
This PR does not change the resulting docker image?
I do not understand.

File: .devcontainer/Dockerfile (hunk 1 above)
Comment (mdsanima) [reply .devcontainer/Dockerfile:5]: This is form official documentation on [Dockerfile](https://github.com/microsoft/vscode-dev-containers/blob/v0.245.2/containers/python-3/.devcontainer/Dockerfile#L6C1-L7C144).

Comment (Finii) [reply .devcontainer/Dockerfile:5]: Well that is just a template. As the commended out lines further down this is intended as a starting point for users that want to have `node` and can easily set a version there - the code is use-ready.

But here, we have a concrete Dockerfile, and we never ever want `node`, so why have code that does nothing?

Comment (mdsanima) [reply .devcontainer/Dockerfile:5]: You're actually correct, but the overall idea of having this file is to build a development environment for the application. The `Dockerfile` is placed in a dedicated folder `.devcontainer` solely for building the container on Dev Container extension in Visual Studio Code. Then, you can open it inside the Visual Studio Code environment, where you continue working in an isolated setup. But I understand if you don't need this code.

**Pull request closed without merging**